IPython/core/magics/code.py

```diff
@@ -500,7 +500,7 @@ def _edit_macro(self,mname,macro):
 
     @skip_doctest
     @line_magic
-    def edit(self, parameter_s='',last_call=['','']):
+    def edit(self, parameter_s='', last_call=None):
```

Comment:
This one is deliberate, and your change will break it. I know it's not considered best practice, but mutable parameters work as a sneaky way to have persistent storage between calls to the same function, so we need to be really careful when 'cleaning up' something like this.